frontend/src/modules/machinery/components/Machinery.css

```diff
@@ -1,5 +1,6 @@
 .machinery {
     line-height: 22px;
+    border-bottom: 2px solid #4D5967;
```

Comment:
That will add the border at the bottom of the section instead of under the search box. Please remove.

Reply:
Not able to decide what border-color to be given as the color of search box is changed